tests/integration/test_cli.sh

```diff
@@ -30,5 +30,30 @@ do
    fi
 done
 
-exit $rc
 
+# Now some specific tests that do not behave like the other binaries
+
+# For BDDI and ARC
+
+ldapsearch --help 2>&1 >/dev/null | grep -q "LDAP";
```

Comment:
maybe `"LDAP"` is not the best test string to test, try rather something longer or more unique from the help output

Reply:
ok changed